.actor/INPUT_SCHEMA.json

```diff
@@ -0,0 +1,90 @@
+            "editor": "textfield"
+        },
+        "datasetId": {
+            "title": "Dataset ID",
+            "type": "string",
+            "description": "Dataset of the ID to download the data from",
+            "editor": "textfield"
+        },
+        "tableName": {
+            "title": "Fully-qualified table name",
+            "type": "string",
+            "description": "Table name in the format DATABASE.SCHEMA.TABLENAME",
+            "editor": "textfield"
+        },
+        "overwrite": {
+            "title": "Overwrite table",
+            "type": "boolean",
+            "description": "Whether to drop the table's data before pushing dataset data into it",
+            "editor": "checkbox",
+            "default": false
+        },
+        "flattenJson": {
+            "title": "Flatten JSON",
+            "type": "boolean",
+            "description": "If you select this option, instead of {a: {b: 1, c: [1, 2]}} you'll get {a.b: 1, a.c.0: 1, a.c.1: 2}",
+            "editor": "checkbox",
+            "default": false
+        },
+        "dataLossConfirmation": {
+            "title": "Confirm data loss",
+            "type": "boolean",
+            "default": false,
+            "description": "If you choose to synchronize your schema or overwrite previous data, check this checkbox to confirm that you agree to lose your previous data",
+            "editor": "checkbox"
+        },
+        "username": {
+            "title": "Connection username",
+            "type": "string",
+            "description": "Your acount's username",
+            "sectionCaption": "Connection",
+            "sectionDescription": "Connection settings",
+            "isSecret": true,
+            "editor": "textfield"
+        },
+        "account": {
+            "title": "Account name",
+            "type": "string",
+            "description": "Account name. You can get it from the url, it's usually a set of letters and is not the same as your username.",
+            "isSecret": true,
+            "editor": "textfield"
+        },
+        "password": {
+            "title": "Password",
+            "type": "string",
+            "description": "Your account's password",
+            "isSecret": true,
+            "editor": "textfield"
+        },
+        "database": {
+            "title": "Database name",
+            "type": "string",
+            "description": "Your database's name.",
+            "isSecret": true,
+            "editor": "textfield"
+        },
+        "warehouse": {
+            "title": "Warehouse",
+            "type": "string",
+            "description": "Supply your custom warehouse name if you want. The deault works fine as well.",
+            "default": "COMPUTE_WH",
+            "editor": "textfield"
+        }
+    },
+    "required": ["datasetId", "flattenJson", "username", "account", "password", "warehouse", "database", "overwrite", "dataLossConfirmation", "tableName"]
```

Comment:
`flattenJSON` should not be required, the same for `overwrite` and `dataLossConfirmation` (defaults to false in the code).

Reply:
Also, let's not make `datasetId` required so we can run this after default webhook.

Reply:
If I make datasetId not required then how will the actor know where to get the data? I thought we can supply POST payload body with webhooks